fdbclient/NativeAPI.actor.cpp

```diff
@@ -795,6 +796,10 @@ void setNetworkOption(FDBNetworkOptions::Option option, Optional<StringRef> valu
 			if(value.present())
 				networkOptions.traceLogGroup = value.get().toString();
 			break;
+		case FDBNetworkOptions::TRACE_FORMAT:
+			validateOptionValue(value, true);
+			networkOptions.traceFormat = value.get().toString();
```

Comment:
Because network options aren't being set on the network thread, I think we have the option here of throwing an error here if the client gets this wrong and having it be immediately visible. What do you think?

It also makes me think we may wonder if we should log an event in selectTraceFormatter if the formatter name isn't recognized. It could help in situations where it's not clear whether the problem is that you put in the wrong string or that the string you tried to specify didn't actually make it into the process at all (e.g. somewhere in the your deployment system, it's not being made a command line option).

Reply:
I like it. I added a change to make it throw an error here, and also log an event in selectTraceFormatter if the format isn't recognized.